vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -212,11 +210,15 @@ private function update_backoff_factor(): void {
 		$backoff_factor = isset( $this->last_heartbeat['backoff_factor'] ) ? (int) $this->last_heartbeat['backoff_factor'] : 0;
 
 		if ( $backoff_factor >= self::MAX_BACKOFF_FACTOR ) {
-			return;
+			$backoff_factor = self::MAX_BACKOFF_FACTOR;
 		} elseif ( $backoff_factor <= 0 ) {
 			$backoff_factor = 1;
 		} else {
 			$backoff_factor = $backoff_factor * 2;
```

Comment:
mek one liner for me brain:
```suggestion
			$backoff_factor = min( $backoff_factor * 2, self::MAX_BACKOFF_FACTOR );
```

Reply:
`+15 −25` ;p